src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -187,6 +201,7 @@ function ExportAnnotations({
                   onChange={setExportFormat}
                   buttonContent={exportFormat.name}
                   data-testid="export-format-select"
+                  right
```

Comment:
I think we should change this API to something like `alignListbox: "left" | "right"`, as  the meaning of a boolean `right` attribute is not obvious on its own.